src/validations/businessCaseSchema.ts

```diff
@@ -33,6 +33,49 @@ const BusinessCaseValidationSchema = {
     successIndicators: Yup.string().required(
       'Tell us how you will determine whethere or not this effort is successful'
     )
+  }),
+  asIsSolution: Yup.object().shape({
+    asIsSolution: Yup.object().shape({
+      title: Yup.string().required('Enter a title for the "As is" solution'),
+      summary: Yup.string().required('Tell us about the "As is" solution'),
+      pros: Yup.string().required('Tell us about the pros of "As is" solution'),
+      cons: Yup.string().required('Tell us about the cons of "As is" solution'),
+      estimatedLifecycleCost: Yup.object().shape({
+        year1: Yup.array().of(
+          Yup.object().shape({
+            phase: Yup.string().required('Select the type of phase for Year 1'),
```

Comment:
Because each year is an array of phases, each error for phases within that year seem harder to distinguish. I don't think it's a problem now, but it could become an issue in the future if validation is more than "is it blank or not?"

Reply:
I think for screens, it _should_ be visually okay. There should be errors populated directly above the phase.

The question I had was for screen readers. I did add labels that indicated, "Year 2 Phase 1", "Year 2 Phase 2". I'm not sure if that'll be sufficient, but it's something we'll see when we go through 508.